frontend/app/student/run-simulation/[instanceId]/page.tsx

```
@@ -1220,36 +1820,92 @@ ${availablePersonas.map(persona => `• @${persona.name.toLowerCase().replace(/\
+                                  <div className="text-xs text-gray-500 mb-2">Mention everyone in this scene</div>
+                                  {simulationData.current_scene.personas.map((persona) => (
+                                    <div
+                                      key={persona.id}
+                                      className="flex items-center gap-2 p-2 hover:bg-gray-50 rounded cursor-pointer"
+                                      onClick={() => {
+                                        const mentionId = persona.name.toLowerCase().replace(/\s+/g, '_');
+                                        setInput(input.replace(/@[^@]*$/, `@${mentionId} `));
+                                        setShowMentionDropdown(false);
+                                      }}
+                                    >
+                                      <div className="w-6 h-6 bg-gray-200 rounded-full flex items-center justify-center flex-shrink-0">
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)